assets/airports/airportLoadList.json

```diff
@@ -432,7 +432,6 @@
         "level": "hard",
         "name": "Tokyo Narita International Airport",
         "premium": false,
```

Comment:
Comma needs to be removed

Reply:
Yeah did realize that, fixed already. Thanks for the nudge!